### ISSUE TITLE
[popup] Show Field API Names

### ISSUE THREAD

**Issue by ktag3000** [tprouvot/Salesforce-Inspector-reloaded#571]
**Make sure to read the troubleshooting section before creating an issue**

- [x] I've read the [common issues](https://tprouvot.github.io/Salesforce-Inspector-reloaded/troubleshooting/) and the one I'm about to create is not one of those documented.
- [x] I've read the [release note](https://github.com/tprouvot/Salesforce-Inspector-reloaded/blob/releaseCandidate/CHANGES.md) and the one I'm about to create is not already fixed.

**Describe the bug**
When you first open the popup and a record page, the 'Show Field API Names' does not come up until you close and reopen

**To Reproduce**
1. Open popup 
2. Close and reopen

**Expected behavior**
I would expect the option to see API names is there the first time it is opened

**Screenshots**
![image](https://github.com/user-attachments/assets/75eb8873-07e2-4ab6-bf38-6f869fdfab26)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Edge
 -Version 128.0.2739.79 (Official build) (64-bit)

**Additional context**

**Comment by uniquestDev** [comment]
Same story with me. Took me a while to realise I have to close/open the inspector again to see the option.

**Comment by tprouvot** [comment]
Hi guys,
Could you try to open the popup 4 or 5 seconds after the end of the page load ? 
@uniquestDev are you on Edge also?
@ktag3000 
Are you using the chrome extension from Edge browser or the extension from the edge store?

**Comment by uniquestDev** [comment]
Nope, hasn't done the trick for me. I have done several tests with various 'waiting time' options, and it wouldn't show unless I open the extension second time. BTW, it is Edge in Win 11.

**Comment by tprouvot** [comment]
IF some of you are using the beta version, could you tell me if this is still happening ? We implemented a cache for the list of sobject that should improve the recordid context recognition and display the show field api name more consistently

**Comment by ktag3000** [comment]
I'm not using beta but on 1.27 I have not noticed the issue for awhile.
